ecommerce/src/components/footer/footer.tsx

```diff
@@ -1 +1,32 @@
-export {};
+import { Button, Col, Row } from 'antd';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Link } from 'react-router-dom';
+import RsImage from '../../images/icon/rs_school.svg';
+
+const Footer = () => {
+    return (
+        <Row
+            style={{
+                display: 'flex',
+                justifyContent: 'center',
+                backgroundColor: 'rgba(0, 0, 0, 0.5)',
+                padding: 5,
+                position: 'relative',
+                zIndex: 2,
+            }}
+        >
+            <Col
+                className="header__item"
+                style={{
+                    cursor: 'pointer',
+                }}
+            >
+                <Link style={{ transition: 'all 0.5s ease' }} to="https://rs.school/js/" target="_blank">
```

Comment:
a good practice to use one type of transition e.g.  all 0.5s ease for the all components. It mean that you can add this style to * or create a class like 'transition'  to avoid an inline style every time